./bootstrap.sh script is broken

As of 8f552e0f, the `./bootstrap.sh` script appears to be broken:

```shell
$ ./bootstrap.sh

/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/catalog/Alias.java:12: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.NonNull;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/ResourceRef.java:9: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.NonNull;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/ResourceRef.java:10: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.Nullable;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/util/Util.java:55: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.NonNull;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/cli/BuildMixin.java:7: error: package dev.jbang.devkitman does not exist
import dev.jbang.devkitman.Jdk;
                          ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/cli/JdkProvidersMixin.java:6: error: package dev.jbang.devkitman does not exist
import dev.jbang.devkitman.JdkManager;
                          ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/Project.java:10: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.NonNull;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/Project.java:11: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.Nullable;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/Project.java:15: error: package dev.jbang.devkitman does not exist
import dev.jbang.devkitman.Jdk;
                          ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/Project.java:16: error: package dev.jbang.devkitman does not exist
import dev.jbang.devkitman.JdkManager;
                          ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/cli/Jdk.java:16: error: package dev.jbang.devkitman does not exist
import dev.jbang.devkitman.JdkManager;
                          ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/cli/Jdk.java:17: error: package dev.jbang.devkitman does not exist
import dev.jbang.devkitman.JdkProvider;
                          ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/Source.java:10: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.NonNull;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/SourceSet.java:10: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.NonNull;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/DocRef.java:5: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.NonNull;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/DocRef.java:6: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.Nullable;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/Project.java:48: error: cannot find symbol
        private JdkManager jdkManager;
                ^
  symbol:   class JdkManager
  location: class Project
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/Project.java:54: error: package Jdk does not exist
        private Jdk.InstalledJdk projectJdk;
                   ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/ProjectBuilder.java:25: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.NonNull;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/ProjectBuilder.java:38: error: package dev.jbang.devkitman does not exist
import dev.jbang.devkitman.JdkManager;
                          ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/Project.java:267: error: cannot find symbol
        public void setJdkManager(JdkManager jdkManager) {
                                  ^
  symbol:   class JdkManager
  location: class Project
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/Project.java:288: error: cannot find symbol
        public JdkManager projectJdkManager() {
               ^
  symbol:   class JdkManager
  location: class Project
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/Project.java:295: error: package Jdk does not exist
        public Jdk.InstalledJdk projectJdk() {
                  ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/BuildContext.java:6: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.NonNull;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/dependencies/ModularClassPath.java:21: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.NonNull;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/dependencies/ModularClassPath.java:23: error: package dev.jbang.devkitman does not exist
import dev.jbang.devkitman.Jdk;
                          ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/dependencies/ModularClassPath.java:69: error: cannot find symbol
        public List<String> getAutoDectectedModuleArguments(@NonNull Jdk jdk) {
                                                                     ^
  symbol:   class Jdk
  location: class ModularClassPath
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/dependencies/ModularClassPath.java:134: error: cannot find symbol
        protected boolean supportsModules(Jdk jdk) {
                                          ^
  symbol:   class Jdk
  location: class ModularClassPath
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/dependencies/MavenCoordinate.java:7: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.NonNull;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/dependencies/MavenCoordinate.java:8: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.Nullable;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/Cache.java:8: error: package dev.jbang.devkitman does not exist
import dev.jbang.devkitman.Jdk;
                          ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/Cache.java:9: error: package dev.jbang.devkitman does not exist
import dev.jbang.devkitman.JdkManager;
                          ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/generators/JarCmdGenerator.java:12: error: package dev.jbang.devkitman does not exist
import dev.jbang.devkitman.Jdk;
                          ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/generators/JshCmdGenerator.java:16: error: package dev.jbang.devkitman does not exist
import dev.jbang.devkitman.Jdk;
                          ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/ProjectBuilder.java:79: error: cannot find symbol
        private JdkManager jdkManager;
                ^
  symbol:   class JdkManager
  location: class ProjectBuilder
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/ProjectBuilder.java:227: error: cannot find symbol
        public ProjectBuilder jdkManager(JdkManager jdkManager) {
                                         ^
  symbol:   class JdkManager
  location: class ProjectBuilder
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/ResourceResolver.java:3: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.NonNull;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/ResourceResolver.java:4: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.Nullable;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/RefTarget.java:11: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.NonNull;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/RefTarget.java:12: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.Nullable;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/TagReader.java:21: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.NonNull;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/cli/JdkProvidersMixin.java:17: error: cannot find symbol
        private JdkManager jdkMan;
                ^
  symbol:   class JdkManager
  location: class JdkProvidersMixin
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/cli/JdkProvidersMixin.java:19: error: cannot find symbol
        protected JdkManager getJdkManager() {
                  ^
  symbol:   class JdkManager
  location: class JdkProvidersMixin
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/cli/Info.java:29: error: package dev.jbang.devkitman does not exist
import dev.jbang.devkitman.Jdk;
                          ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/cli/Info.java:30: error: package dev.jbang.devkitman does not exist
import dev.jbang.devkitman.JdkManager;
                          ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/util/JavaUtil.java:14: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.NonNull;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/util/JavaUtil.java:19: error: package dev.jbang.devkitman.jdkinstallers does not exist
import dev.jbang.devkitman.jdkinstallers.FoojayJdkInstaller;
                                        ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/util/JavaUtil.java:21: error: package dev.jbang.devkitman.util does not exist
import dev.jbang.devkitman.util.RemoteAccessProvider;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/cli/App.java:29: error: package dev.jbang.devkitman does not exist
import dev.jbang.devkitman.Jdk;
                          ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/util/JavaUtil.java:26: error: cannot find symbol
        public static JdkManager defaultJdkManager(String... names) {
                      ^
  symbol:   class JdkManager
  location: class JavaUtil
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/util/JavaUtil.java:31: error: cannot find symbol
        public static JdkManager defaultJdkManager(List<String> names) {
                      ^
  symbol:   class JdkManager
  location: class JavaUtil
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/util/JavaUtil.java:182: error: cannot find symbol
        public static String resolveInJavaHome(@NonNull String cmd, @NonNull Jdk jdk) {
                                                                             ^
  symbol:   class Jdk
  location: class JavaUtil
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/util/JavaUtil.java:18: error: package dev.jbang.devkitman does not exist
import dev.jbang.devkitman.*;
^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/util/JavaUtil.java:20: error: package dev.jbang.devkitman.jdkproviders does not exist
import dev.jbang.devkitman.jdkproviders.*;
^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/dependencies/ArtifactResolver.java:44: error: package eu.maveniverse.maven.mima.context does not exist
import eu.maveniverse.maven.mima.context.Context;
                                        ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/dependencies/ArtifactResolver.java:45: error: package eu.maveniverse.maven.mima.context does not exist
import eu.maveniverse.maven.mima.context.ContextOverrides;
                                        ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/dependencies/ArtifactResolver.java:46: error: package eu.maveniverse.maven.mima.context does not exist
import eu.maveniverse.maven.mima.context.Runtimes;
                                        ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/dependencies/ArtifactResolver.java:49: error: cannot find symbol
        private final Context context;
                      ^
  symbol:   class Context
  location: class ArtifactResolver
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/buildsteps/NativeBuildStep.java:12: error: package dev.jbang.devkitman does not exist
import dev.jbang.devkitman.Jdk;
                          ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/buildsteps/NativeBuildStep.java:96: error: cannot find symbol
        private static String resolveInGraalVMHome(String cmd, Jdk jdk) {
                                                               ^
  symbol:   class Jdk
  location: class NativeBuildStep
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/resolvers/AliasResourceResolver.java:6: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.NonNull;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/resolvers/AliasResourceResolver.java:7: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.Nullable;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/resolvers/ClasspathResourceResolver.java:7: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.NonNull;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/InputStreamResourceRef.java:8: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.NonNull;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/resolvers/CombinedResourceResolver.java:8: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.NonNull;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/resolvers/FileResourceResolver.java:10: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.NonNull;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/resolvers/FileResourceResolver.java:11: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.Nullable;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/resolvers/GavResourceResolver.java:5: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.NonNull;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/resolvers/LiteralScriptResourceResolver.java:13: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.NonNull;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/resolvers/RemoteResourceResolver.java:12: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.NonNull;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/resolvers/RenamingScriptResourceResolver.java:10: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.NonNull;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/resolvers/SiblingResourceResolver.java:8: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.NonNull;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/sources/GroovySource.java:12: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.NonNull;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/sources/JavaSource.java:11: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.NonNull;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/sources/JavaSource.java:13: error: package dev.jbang.devkitman does not exist
import dev.jbang.devkitman.Jdk;
                          ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/sources/JshSource.java:5: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.NonNull;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/sources/KotlinSource.java:11: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.NonNull;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/sources/MarkdownSource.java:9: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.NonNull;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/CodeBuilderProvider.java:7: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.NonNull;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/spi/IntegrationManager.java:30: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.NonNull;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/spi/IntegrationManager.java:38: error: package dev.jbang.devkitman does not exist
import dev.jbang.devkitman.Jdk;
                          ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/spi/IntegrationManager.java:220: error: cannot find symbol
                        Jdk jdk)
                        ^
  symbol:   class Jdk
  location: class IntegrationManager
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/util/JarUtil.java:14: error: package dev.jbang.devkitman does not exist
import dev.jbang.devkitman.Jdk;
                          ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/util/JarUtil.java:20: error: cannot find symbol
        public static void createJar(Path jar, Path src, Manifest manifest, String mainClass, Jdk jdk)
                                                                                              ^
  symbol:   class Jdk
  location: class JarUtil
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/util/JarUtil.java:25: error: cannot find symbol
        public static void updateJar(Path jar, Manifest manifest, String mainClass, Jdk jdk)
                                                                                    ^
  symbol:   class Jdk
  location: class JarUtil
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/util/JarUtil.java:30: error: cannot find symbol
        private static void runJarCommand(Path jar, String action, Path src, Manifest manifest, String mainClass, Jdk jdk)
                                                                                                                  ^
  symbol:   class Jdk
  location: class JarUtil
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/util/JarUtil.java:68: error: cannot find symbol
        private static void runJarCommand(List<String> arguments, Jdk jdk) throws IOException {
                                                                  ^
  symbol:   class Jdk
  location: class JarUtil
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/util/JavaUtil.java:35: error: package JdkManager does not exist
        public static class JdkManBuilder extends JdkManager.Builder {
                                                            ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/util/JavaUtil.java:41: error: package JdkManager does not exist
                public JdkManager.Builder provider(String... names) {
                                 ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/util/JavaUtil.java:45: error: package JdkManager does not exist
                public JdkManager.Builder provider(List<String> names) {
                                 ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/util/JavaUtil.java:56: error: cannot find symbol
                public JdkManager build() {
                       ^
  symbol:   class JdkManager
  location: class JdkManBuilder
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/util/JavaUtil.java:69: error: cannot find symbol
                private JdkProvider createProvider(String providerName) {
                        ^
  symbol:   class JdkProvider
  location: class JdkManBuilder
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/util/JavaUtil.java:94: error: cannot find symbol
        static class JBangRemoteAccessProvider implements RemoteAccessProvider {
                                                          ^
  symbol:   class RemoteAccessProvider
  location: class JavaUtil
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/util/ModuleUtil.java:15: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.Nullable;
                               ^
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/ResourceRef.java:30: error: cannot find symbol
        @Nullable
         ^
  symbol:   class Nullable
  location: interface ResourceRef
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/ResourceRef.java:64: error: cannot find symbol
        static boolean isStdin(@NonNull String scriptResource) {
                                ^
  symbol:   class NonNull
  location: interface ResourceRef
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/ResourceRef.java:78: error: cannot find symbol
        @NonNull
         ^
  symbol:   class NonNull
  location: interface ResourceRef
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/ResourceRef.java:112: error: cannot find symbol
        @NonNull
         ^
  symbol:   class NonNull
  location: interface ResourceRef
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/ResourceRef.java:127: error: cannot find symbol
        static ResourceRef forFile(@NonNull Path file) {
                                    ^
  symbol:   class NonNull
  location: interface ResourceRef
/home/vorburger/git/github.com/jbangdev/jbang/src/main/java/dev/jbang/source/ResourceRef.java:141: error: cannot find symbol
        static ResourceRef forResolvedResource(@NonNull String resource, @NonNull Path resolvedResource) {
                                                ^
  symbol:   class NonNull
  location: interface ResourceRef
100 errors
only showing the first 100 errors, of 256 total; use -Xmaxerrs if you would like to see more
[jbang] [2:202] Deleting folder /home/vorburger/.jbang/cache/jars/Main.java.a0395c4e332758584e8baa4e4fa913c70130a717733bc1eec9ab68b9ef3194d0/classes
[jbang] [2:207] Deleting folder /home/vorburger/.jbang/cache/jars/Main.java.a0395c4e332758584e8baa4e4fa913c70130a717733bc1eec9ab68b9ef3194d0/generated
[jbang] [2:207] [ERROR] Error during compile
dev.jbang.cli.ExitException: Error during compile
        at dev.jbang.source.buildsteps.CompileBuildStep.runCompiler(CompileBuildStep.java:135)
        at dev.jbang.source.buildsteps.CompileBuildStep.runCompiler(CompileBuildStep.java:120)
        at dev.jbang.source.buildsteps.CompileBuildStep.compile(CompileBuildStep.java:69)
        at dev.jbang.source.buildsteps.CompileBuildStep.build(CompileBuildStep.java:54)
        at dev.jbang.source.buildsteps.CompileBuildStep.build(CompileBuildStep.java:39)
        at dev.jbang.source.AppBuilder.build(AppBuilder.java:97)
        at dev.jbang.source.AppBuilder.build(AppBuilder.java:22)
        at dev.jbang.cli.BaseExportCommand.doCall(Export.java:67)
        at dev.jbang.cli.BaseCommand.call(BaseCommand.java:149)
        at dev.jbang.cli.BaseCommand.call(BaseCommand.java:24)
        at picocli.CommandLine.executeUserObject(CommandLine.java:2041)
        at picocli.CommandLine.access$1500(CommandLine.java:148)
        at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2453)
        at dev.jbang.cli.JBang$3.handle(JBang.java:157)
        at dev.jbang.cli.JBang$3.handle(JBang.java:152)
        at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273)
        at picocli.CommandLine$RunLast.execute(CommandLine.java:2417)
        at picocli.CommandLine.execute(CommandLine.java:2170)
        at dev.jbang.Main.main(Main.java:27)
```

--- a/bootstrap.sh
+++ b/bootstrap.sh
@@ -2,9 +2,9 @@
 
 ## in gradle we have plugin to generate BuildConfig with version info in it.
 ## For now just generate a placeholder. Ultimately could be some other script to generate the file.
-mkdir -p generated
+mkdir -p generated/
 cat << EOF > generated/BuildConfig.java
-package dev.jbang;
+package dev.jbang.util;
 
 public final class BuildConfig {
     public static final String VERSION = "999-jbang";
@@ -15,4 +15,15 @@ public final class BuildConfig {
 }
 EOF
 
-jbang export local --verbose --force -O jbang.jar --sources 'src/main/java,generated/' --files 'src/main/resources' --repos mavencentral,jitpack --deps org.jboss:jandex:2.2.3.Final,org.slf4j:slf4j-nop:1.7.30,com.offbytwo:docopt:0.6.0.20150202,org.apache.commons:commons-text:1.10.0,org.apache.commons:commons-compress:1.20,info.picocli:picocli:4.6.3,io.quarkus.qute:qute-core:1.12.2.Final,kr.motd.maven:os-maven-plugin:1.7.0,org.codehaus.plexus:plexus-java:1.0.6,com.google.code.gson:gson:2.9.0,org.jsoup:jsoup:1.13.1,org.codejive:java-properties:0.0.4,com.github.jbangdev.jbang-resolver:shrinkwrap-resolver-api:3.1.5-allowpom,com.github.jbangdev.jbang-resolver:shrinkwrap-resolver-impl-maven:3.1.5-allowpom src/main/java/dev/jbang/Main.java
+# Use Gradle to get the implementation dependencies in a clean format
+# this is done just to avoid having to double maintain the list of dependencies
+DEPS=$(./gradlew dependencies --configuration compileClasspath --quiet | \
+       grep -E '^[+\\-]---' | \
+       sed -E 's/^[+\\-]+---[ \\]*//' | \
+       sed -E 's/ -> .*$//' | \
+       sed 's/ (\*)//' | \
+       grep -E '^[^:]+:[^:]+:[^:]+$' | \
+       sort -u | \
+       tr '\n' ',' | sed 's/,$//')
+
+jbang export local --force -O jbang.jar --sources src/main/java,src/main/java9,generated --files src/main/resources --repos mavencentral --deps "$DEPS" src/main/java/dev/jbang/Main.java

